### ISSUE TITLE
More Settings For Surround

### ISSUE THREAD

**Issue by VoidCyborg** [THEREALWWEFAN231/Inertia#875]
**Minecraft Version(s)**
All
**What is the suggestion**
Add in surround settings to close face by obsidian. Like it closes legs now.

**Why would this feature be useful**
Because it would be helpful against face spam(by crystals).

**Comment by feliscatusmeows** [comment]
ooh so like a auto-self trap?

**Comment by VoidCyborg** [comment]
Yeah like auto-self trap, becouse now in pvp enemies always use face spam(crystals) or anvils and place a lot of them on botom block, but if surrounw will place block there it would be helpful.
